includes/Upgrade/Upgrades/V_3_6_3.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WeDevs\Dokan\Upgrade\Upgrades;
+
+use WeDevs\Dokan\Abstracts\DokanUpgrader;
+use WeDevs\Dokan\ProductCategory\Categories;
+
+class V_3_6_3 extends DokanUpgrader {
+
+    /**
+     * Updates categories for vendor subscription products.
+     *
+     * @since DOKAN_SINCE
+     *
+     * @return void
+     */
+    public static function update_vendor_subscription_products_categories() {
+        $args = [
+            'type'   => 'product_pack',
+            'status' => 'any',
+            'return' => 'ids',
+            'limit'  => -1,
+        ];
+
+        $products = wc_get_products( $args );
+        $dokan_category = new Categories();
+        $all_categories = $dokan_category->get_all_categories( true );
+
+        foreach ( $products as $product_id ) {
+            $categories = get_post_meta( $product_id, '_vendor_allowed_categories', true );
```

Comment:
priority will be vendor user meta, then post meta, so I think we need to rethink this approach